karma.conf.js

```diff
@@ -31,7 +31,7 @@ module.exports = function(config) {
     autoWatch: true,
     singleRun: true,
     concurrency: 1,
-    browsers: ["Chrome", "Firefox", "Safari"],
+    browsers: ["Chrome"],
```

Comment:
Oops, I should probably revert this back. I'm always modifying it locally so that Karma doesn't wait for missing browsers.